### DISCUSSION TITLE
Is there an easy way to insert special characters such as emphasis (&#769;)

### DISCUSSION THREAD

**Discussion by glorious-web-developer** [ueberdosis/tiptap#820] [General Questions]
It's pretty much all in the title. i was trying to implement a feature close to find/replace to add emphasis for a button. But when i tried replacing it with a special HTML character such as emphasis (`&#769;`) it replaced it with characters(`o&#769;`) and not an emphasized(ó) letter. Got to mention that if special characters are present in content prop before initialization, it works fine. What am i missing here?
Thanks in advance and sorry if i missed a similar question.

**Comment by hanspagel** [comment]
How did you try to replace it with a special HTML character?

**Reply by glorious-web-developer** [reply]
I was trying find/replace feature like in an example here https://tiptap.dev/search-and-replace
placed `o` in find and `o&#769;` in replace

**Reply by hanspagel** [reply]
Hm, I can’t reproduce the issue you’re describing. You need to help me understand the issue better.

I went to the example, searched for `o` and replaced all with `ó`, which worked fine at first glance.
![Hthq5unuN6](https://user-images.githubusercontent.com/1577992/91971866-2db52400-ed1a-11ea-857f-e27f83c38d38.png)

**Reply by glorious-web-developer** [reply]
It is, when you're replacing it with special character itself, but if you enter `o&#769;`, which should be transpiled into special character (ó) it displays exactly `o&#769;`. 
![Screenshot from 2020-09-03 10-32-18](https://user-images.githubusercontent.com/70586583/92085009-e2118180-edd0-11ea-9922-49461c5efe38.png)

**Reply by hanspagel** [reply]
It’s not expected that any characters or combination of characters should be converted to other characters. That would lead to a ton of issues, e. g. what’s with people wanting to write about the actual `&#769;`?

**Reply by glorious-web-developer** [reply]
I get that, but is there a way to manually convert or force to convert inserted special symbols?
